.github/workflows/bundle_with_dakota_caller.yml

```diff
@@ -0,0 +1,102 @@
+name: 🏎️ Make & Test Wheels 🏎️
+
+env: 
+  ERT_SHOW_BACKTRACE: 1
+  NO_PROJECT_RES: 1
+  BOOST_VERSION: 1.83.0
+  DAKOTA_VERSION: 6.18.0
+  SEBA_TAG: 6.12.0
+  INSTALL_DIR: local
+  FORCE_REBUILD: false
```

Comment:
Not used anymore?

Reply:
Forgot this one, oops. A bit because I think we'll have to revise the workflow dispatch, then we might want to add it, but it does not make sense to just leave it here "for the future" indeed. Removing

Reply:
Agreed.